### DISCUSSION TITLE
stopping font cache being created when using convert

### DISCUSSION THREAD

**Discussion by timsoftgit** [ImageMagick/ImageMagick#6401] [Help]
### ImageMagick version

7.1.0-22

### Operating system, version and so on

Slackware 15.0 64bit

### Description

is there a way to prevent fontcache being created when convert is run, or at least specify where it is created.? At the moment, when converting svg images it is creating a font cache file in /var/cache/fontconfig/ (even if the svg doesn't appear to conatin a font). convert is being run in a makefile as part of package creation, and I want to avoid things being written outside the package tree. I did try setting the FONTCONFIG_FILE environment variable to a local file where the local file contains a local location for cachedir, but it didn't appear to help.

**Comment by urban-warrior** [comment]
See `file:///usr/share/doc/fontconfig/fontconfig-user.html`.

**Comment by timsoftgit** [comment]
i should mention this behaviour only seems to happen when inkscape is installed

**Comment by timsoftgit** [comment]
i found a work-around. create a local (to the build system) inkscape executeable containing
#!/bin/sh
exit 1
and temporarily add the local path to the front of the PATH environment variable.
then when running  imagemagik's convert it trys running inkscape and fails, and it will use the rsvgconvert instead.
